### PR TITLE
explore pages created successfully

### PR DESCRIPTION
**Pull request by ivinayakg**

### CODE REVIEW

File: src/components/CarouselRow/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  });
+  const row = useRef();
+
+  const resizeHandler = (e) => {
+    if (row.current.offsetWidth < window.screen.width) {
+      setShowButtons({ left: true, right: true, verified: true });
+    } else {
+      setShowButtons({ left: false, right: false, verified: true });
+    }
+  };
```
Comment (rahulyadav139) [inline src/components/CarouselRow/index.js:20]: If you are not using the event then you can remove "e" argument.

File: src/components/VideoCard.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    videoRef.current.currentTime = 20;
+  }, []);
+
+  const changeVideoFrameOnHover = (e) => {
+    if (e.type !== "mouseleave") {
+      setPlay(true);
+    } else {
+      setPlay(false);
+    }
+  };
```
Comment (rahulyadav139) [inline src/components/VideoCard.js:31]: you can use ES6 template literal here.

**Review by rahulyadav139 — APPROVED** [review]

File: src/components/VideoCard.js (hunk 2 above)
Comment (ivinayakg) [reply src/components/VideoCard.js:31]: right

**Review by sameep-s — APPROVED** [review]
Code looks great
Things that i liked:

- neat and clean approach.
- good naming convention.

**Pull request merged**